packages/wmr/src/lib/compile-single-module.js

```diff
@@ -74,6 +74,7 @@ export const compileSingleModule = withCache(
 			...ROLLUP_FAST_OUTPUT,
 			dir: out,
 			assetFileNames: '[name].[ext]',
+			...(typeof filename !== 'undefined' && { entryFileNames: filename }),
```

Comment:
So I attempted to reuse `assetFileNames` here, but for whatever reason, it always created an asset with an extension of `.js`, even if `.mjs` was passed to it. `entryFileNames` seems to handle this though.

I cannot rightly say I understand the difference between the two and the Rollup docs don't really help in that regard. Someone who better understands might want to look at this.

Reply:
This _seems_ like it should work, and passes the test suite, but I don't understand the difference between the two enough to know for sure whether it's acceptable.

```suggestion
			entryFileNames: filename || '[name].[ext]',
```

Reply:
@rschristian that suggestion makes sense - they're doing the same thing really, since `filename` didn't exist before.

Reply:
Good to know!

Didn't know if there was enough of a difference between an asset and an entry, even for this tiny, single file use case that would be an issue.